sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -38,8 +40,33 @@ class PlaygroundActivity : AppCompatActivity() {
         setContentView(R.layout.activity_playground)
 
         updateIndicators()
+        setupDonut()
         initControls()
-        Handler().postDelayed({ fillInitialData() }, 500)
+        Handler().postDelayed({
+            fillInitialData()
+            runInitialAnimation()
+        }, 800)
+    }
+
+    private fun setupDonut() {
+        donut_view.cap = 5f
+        donut_view.masterProgress = 0f
+        donut_view.alpha = 0f
+    }
+
+    private fun runInitialAnimation() {
+        ValueAnimator.ofFloat(0f, 1f).apply {
+            duration = 1000
+            interpolator = FastOutSlowInInterpolator()
+            addUpdateListener {
+                donut_view.masterProgress = it.animatedValue as Float
+                donut_view.alpha = it.animatedValue as Float
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3317dfecd010d9a5776f0a2a2e33736acb814042/sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt#L63"></span>it.animatedValue cannot be safely cast to Float.</td>
    </tr>
  </tbody>
</table>